packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec

```diff
@@ -71,9 +48,10 @@ cp -a .%{gem_dir}/* \
 
 %files
 %dir %{gem_instdir}
+%exclude %{gem_instdir}/.github
 %exclude %{gem_instdir}/.gitignore
-%exclude %{gem_instdir}/.github/
 %exclude %{gem_instdir}/Appraisals
+%doc %{gem_instdir}/CHANGES.md
```

Comment:
This moves from doc files to regular files. Probably not what we want.

Reply:
probably not, but it happens with every package that has a `CHANGES.md` and I have not the slightest idea why (the file is correctly tagged as `doc` so it should not happen, and yet…)